package.json

```diff
@@ -31,14 +31,14 @@
     "date-fns": "^2.23.0",
     "next": "^12.0.4",
     "next-svgr": "^0.0.2",
-    "next-tinacms-cloudinary": "^3.5.11",
+    "next-tinacms-cloudinary": "^3.5.23",
```

Comment:
This is an oddity.  I don't think we've made any recent changes to this package, so `tina-cloud-starter` must've just been using an older version? 🤷